lib/matplotlib/collections.py

```diff
@@ -1254,15 +1300,21 @@ class AsteriskPolygonCollection(RegularPolyCollection):
 
 
 class LineCollection(Collection):
-    """
-    All parameters must be sequences or scalars; if scalars, they will
-    be converted to sequences.  The property of the ith line
-    segment is::
+    r"""
+    Represents a sequence of `~.lines.Line2D`\s that should be drawn together.
 
-       prop[i % len(props)]
+    This class extends `.Collection` to represent a sequence of
+    `~.lines.Line2D`\s instead of just a sequence of `~.patches.Patch`\s.
```

Comment:
```suggestion
    `.Line2D`\s instead of just a sequence of `.Patch`\es.
```

Maybe a bit nitpicky, but `Line2D` and `Patch` are unique classes within Matplotlib, so we don't need prefixing with the module. And not doing so increases readability of the plain-text docstring.